crates/net/network/src/manager.rs

```diff
@@ -537,9 +542,11 @@ where
                 }
                 SwarmEvent::IncomingTcpConnection { remote_addr, session_id } => {
                     trace!(target : "net", ?session_id, ?remote_addr, "Incoming connection");
+                    this.metrics.incoming_connections.increment(1);
```

Comment:
this does not seem to be decremented ever, imo the metric is more useful if it shows you how many at any point in time are incoming vs outgoing

Reply:
this makes sense, I guess both would be useful, `total_incoming` and `incoming`?

Reply:
total_incoming can be derived more easily, just having one is fine